### ISSUE TITLE
Fine-tune the model

### ISSUE THREAD

**Issue by YueYANG1996** [antoine77340/S3D_HowTo100M#11]
Hi Antoine,

If I want to fine-tune this model, what's the format of training examples? Do you have a sample code for fine-tuning? Thanks!

Best,
Yue

**Comment by antoine77340** [comment]
You could see the training code here: https://github.com/antoine77340/MIL-NCE_HowTo100M
(that's not fine-tuning but just training from scratch, I guess the only difference would be to start with a lower learning rate if you finetune)

**Comment by YueYANG1996** [comment]
Okay, got it, thanks!
